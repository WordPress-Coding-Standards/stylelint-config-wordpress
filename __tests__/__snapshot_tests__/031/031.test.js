'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '031', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '031' )],
		configFile: systemTestUtils.caseConfig( '031' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
