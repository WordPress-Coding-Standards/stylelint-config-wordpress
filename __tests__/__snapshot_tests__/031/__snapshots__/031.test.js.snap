// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`031 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "031/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 43,
        "rule": "no-missing-end-of-source-newline",
        "severity": "error",
        "text": "Unexpected missing end-of-source newline (no-missing-end-of-source-newline)",
      },
      Object {
        "column": 1,
        "line": 11,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 16,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 20,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 23,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 26,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 30,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 40,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 30,
        "line": 36,
        "rule": "selector-pseudo-class-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown pseudo-class selector \\":first\\" (selector-pseudo-class-no-unknown)",
      },
    ],
  },
]
`;
